src/didactWebView.ts

```diff
@@ -203,6 +208,10 @@ export class DidactWebviewPanel {
 		DidactWebviewPanel.postNamedSimpleMessage("returnCommands");
 	}
 
+	public setActiveContext(value: boolean) {
+		vscode.commands.executeCommand('setContext', 'didact.webview', value);
```

Comment:
executeCommand is returning a Thenable means its executed asynchronously. Maybe worth to check if we need to async/await it or not.

Reply:
All the examples I saw had it not async -- but it would definitely have some serious ripples if I change it to async. I'm going to leave it unless it becomes a problem.

Reply:
I looked into this a bit and it would be a nightmare of ripples -- but even in vscode's own code they're not treating it async - https://github.com/Microsoft/vscode/blob/cd7c7c5fa2c16c6e8281436c35e9a7709cfbd89d/extensions/markdown-language-features/src/features/previewManager.ts#L152